src/components/emote-picker/EmotePicker.jsx

```diff
@@ -3,7 +3,19 @@ import { EMOTES } from '../../const/emotes';
 import Emote from './emote/Emote';
 import './emotePicker.scss';
 
-const EmotePicker = ({ onClick }) => {
+const EmotePicker = ({ searchString, onClick }) => {
+  console.log(searchString);
+
+  if (searchString) {
```

Comment:
Looks like a bit of a code duplication here.

Both of your if-else-branches are mostly doing the same thing.
```
EMOTES
.filter((emote) => !searchString || emote.tags[0].toLowerCase().includes(searchString.toLowerCase())).`
.map(...)
```